styles/style.css

```diff
@@ -0,0 +1,182 @@
+body {
+    box-sizing: border-box;
+    background-color: #DCDCDC;
+}
+.header {
+    background-image: url("../images/header-image.jpg");
+    background-color: #003C63;
+    background-size: 100%;
+    height: 15rem;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+.caption {
+    display: block;
+    color: #42aa04;
+    font-style: italic;
+    line-height: 10%;
+}
+.checkout {
+    color: #FFFFFF;
+    background-color: #42aa04 ;
+}
+.header-box-1 {
+    text-align: center;
+    margin: 2rem 2rem 6rem 4rem;
+}
+.header-box-2  {
+    display: block; 
+    margin: 2rem 6rem 10rem 20rem;
```

Comment:
this margin is kind of a heavy handed way to do this. Maybe you could cut it and use some different flex box and padding code?